examples/base/package.json

```diff
@@ -0,0 +1,38 @@
+{
+  "name": "@raincatcher/example-base",
```

Comment:
We can publish placeholder for this module to avoid someone using `@raincatcher/example-base` for their code :D